app-tests/run.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+set -e
+
+export OPAL_AUTH_PUBLIC_KEY
+export OPAL_AUTH_PRIVATE_KEY
+export OPAL_AUTH_MASTER_TOKEN
+export OPAL_CLIENT_TOKEN
+export OPAL_DATA_SOURCE_TOKEN
+
+function generate_opal_keys {
+  echo "- Generating OPAL keys"
+
+  ssh-keygen -q -t rsa -b 4096 -m pem -f opal_crypto_key -N ""
+  OPAL_AUTH_PUBLIC_KEY="$(cat opal_crypto_key.pub)"
+  OPAL_AUTH_PRIVATE_KEY="$(tr '\n' '_' < opal_crypto_key)"
+  rm opal_crypto_key.pub opal_crypto_key
+
+  OPAL_AUTH_MASTER_TOKEN="$(openssl rand -hex 16)"
+  OPAL_AUTH_JWT_AUDIENCE=https://api.opal.ac/v1/ OPAL_AUTH_JWT_ISSUER=https://opal.ac/ OPAL_REPO_WATCHER_ENABLED=0 opal-server run &
+  sleep 2;
```

Comment:
why 2 ? 
would make this configurable

Reply:
I don't know, it works. That's how `run-example-with-security.sh` already works for ages so I wouldn't trip about it